Lib/sysconfig.py

```diff
@@ -667,8 +667,18 @@ def get_platform():
 
     """
     if os.name == 'nt':
-        if 'GCC' in sys.version:
-            return 'mingw'
+        if 'gcc' in sys.version.lower():
+            if 'ucrt' in sys.version.lower():
+                if 'amd64' in sys.version.lower():
+                    return 'mingw_x86_64_ucrt'
+                return 'mingw_i686_ucrt'
+            if 'clang' in sys.version.lower():
+                if 'amd64' in sys.version.lower():
+                    return 'mingw_x86_64_clang'
+                return 'mingw_i686_clang'
```

Comment:
ditto...
```suggestion
                if '(arm64)' in sys.version.lower():
                    return 'mingw_aarch64_clang'
                return 'mingw_i686_clang'
```

???  arm32 isn't built here, so don't know if you want to handle that for completeness or not.